include/drivers/led_strip/tlc5971.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 Esco Medical ApS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_LED_STRIP_TLC5071_H_
+#define ZEPHYR_INCLUDE_DRIVERS_LED_STRIP_TLC5071_H_
+
+
+/**
+ * @brief Maximum value for global brightness control, i.e 100% brightness
+ */
+#define TLC5971_GLOBAL_BRIGHTNESS_CONTROL_MAX 127
+
+/**
+ * @brief Set the global brightness control levels for the tlc5971 strip.
+ *
+ * change will take effect on next update of the led strip
+ *
+ * @param dev LED strip device
+ * @param pixel global brightness values for RGB channels
+
+ * @return 0 on success, negative on error
+ */
+int tlc5971_set_global_brightness(const struct device *dev, struct led_rgb pixel);
```

Comment:
Since this function can be called from userland, there is maybe a Zephyr API to use (like system calls).

Just to be sure, let's ask @mbolivar-nordic to review it again.

Reply:
From what I can tell from other drivers, there should be no special considerations here regarding usermode, but I cannot say for sure.

Reply:
Right, the entire led_strip API lacks userspace support, so there's nothing to do here.

Reply:
Maybe this was discussed before, can this function be integrated as a part of the API? This also reminds me about unifying `led_strip` and `led` at some point!

Reply:
OKay, fine with me.